src/Browscap/Generator/DirectoryMissingException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+declare(strict_types = 1);
+namespace Browscap\Generator;
+
+class DirectoryMissingException extends \InvalidArgumentException
```

Comment:
You have this exception class, and "NoDirectoryException" below, don't they mean the same thing?

Reply:
the other exception is renamed